formats/dot/internal/errors/errors.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// generated by gocc; DO NOT EDIT.
+// Code generated by gocc; DO NOT EDIT.
```

Comment:
During the PR process, I've manually added the copyright notice to auto-generated files. Not even sure if this makes sense, or if those can be copyrighted in the same way since they are produced by a tool and not us.

If we simply regenerate the files, the "Code generated by" comment should be on top, without a copyright header. This would solve this issue you highlighted, and also facilitate regeneration in the future. Another approach would be to automatically insert the copyright header between the "Code generated by" comment and the `package` keyword.

I would prefer not the first approach. What are your thoughts?